client/my-sites/subscribers/components/empty-list-view/empty-list-view.tsx

```diff
@@ -0,0 +1,50 @@
+import { Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { GrowAudienceIcon, ImportSubscribersIcon, TurnIntoSubscribersIcon } from './icons';
+import './styles.scss';
+
+type EmptyListCTALinkProps = {
+	href: string;
+	icon: JSX.Element;
+	text: string;
+};
+
+const EmptyListCTALink = ( { href, icon, text }: EmptyListCTALinkProps ) => (
+	<a className="empty-list-view__cta-link" href={ href } target="_blank" rel="noreferrer">
+		<span className="empty-list-view__cta-link-icon">{ icon }</span>
+		<span className="empty-list-view__cta-link-text">{ text }</span>
+		<Gridicon icon="chevron-right" size={ 18 } />
+	</a>
+);
+
+const EmptyListView = () => {
+	const translate = useTranslate();
+
+	return (
+		<div className="empty-list-view">
+			<h2 className="empty-list-view__title">{ translate( 'Grow your subscribers' ) }</h2>
+			<p className="empty-list-view__description">
+				{ translate(
+					'Publishing & sharing content can help bring traffic to your site. Let’s help you get started.'
+				) }
+			</p>
+			<EmptyListCTALink
+				href="https://wordpress.com/support/wordpress-editor/blocks/subscribe-block/"
+				icon={ <TurnIntoSubscribersIcon /> }
```

Comment:
I think we could consider using `<Icon icon={ chartBar } />` here (coming from `@wordpress/icons`) instead of defining our own icon component.

```suggestion
				icon={ <Icon icon={ chartBar } /> }
```

Similarly with the other icons below.

Reply:
Fixed, thank you for the feedback Ivan!